src/components/molecules/Visualizer/Widget/Storytelling/index.tsx

```diff
@@ -115,12 +115,10 @@ const Wrapper = styled.div`
   height: 80px;
   width: 500px;
   box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
Any reason why you removed space on this file? Doesn't it make things harder to read? Pretty sure most components keep space between media queries, hovers, etc for clarity.

Reply:
I didn't touch this file but the code formatter did this...
To follow the formatter's config, I thought I should keep this. 🤔